src/simplifier.cpp

```diff
@@ -1058,6 +1058,15 @@ static size_t countTriangles(Triangle* tritable, size_t tritable_size, const Ver
 		VertexID b2 = vertex_ids_pos_only[indices[i + 1]];
```

Comment:
noting that this code might break if the input mesh has a non-multiple-of-3 index_count but that seems to be from the original library

Reply:
At the start of meshopt_simplifySloppy it asserts that index_count is a multiple of 3 :)